db/single/mixed/0/image_list.txt

```diff
@@ -12,3 +12,4 @@ dog-3277414_640.jpg
 422-odd-width.jpg
 444.jpg
 411-non-multiple-4-width.jpg
+2x4.jpg
```

Comment:
is that the "official" name of this chroma sub-sampling mode?

Reply:
No it is not. But I don't know if there is any for it. That is how `identify` reports it -`2x4 ,1x1, 1x1`.